spec/content.spec.js

```diff
@@ -169,6 +169,38 @@ describe('Content', () => {
     })
   })
 
+  describe('getTagsByNameAndAttributes()', () => {
```

Comment:
Please always use `function () {}` instead of `() => {` when using mocha.

Reply:
... I see it's done like this in the whole file, but it is still wrong as it error prone and doesnt properly isolate tests when `() => {}` is used together with `this`. We should quickly fix the whole file.